src/us/kbase/workspace/test/kbase/admin/AdministrationCommandSetInstallerTest.java

```diff
@@ -155,9 +176,50 @@ private void runCommandFail(
 		}
 	}
 	
+	@Test
+	public void installFailNulls() throws Exception {
+		final TestMocks m = initTestMocks();
+		final LocalTypeServerMethods ltsm = (LocalTypeServerMethods) m.tsm;
+		final UserValidator uval = mock(UserValidator.class);
+		final WorkspaceAdministration.Builder b = WorkspaceAdministration.getBuilder(m.ah, uval);
+
+		failInstall(null, m.wsmeth, ltsm, new NullPointerException("builder"));
+		failInstall(null, m.wsmeth, m.del, new NullPointerException("builder"));
+		failInstall(b, null, ltsm, new NullPointerException("wsmeth"));
+		failInstall(b, null, m.del, new NullPointerException("wsmeth"));
+		failInstall(b, m.wsmeth, (LocalTypeServerMethods) null, new NullPointerException("type"));
+		failInstall(b, m.wsmeth, (WorkspaceDelegator) null, new NullPointerException("delegator"));
```

Comment:
presumably if more than one argument is null, one of these errors will be triggered, so there's no point in testing combinations of nulls.

Reply:
yeah, whichever null test comes first will trigger.